rllib/algorithms/algorithm.py

```diff
@@ -871,7 +871,7 @@ def step(self) -> ResultDict:
         results = self._compile_iteration_results(
             episodes_this_iter=episodes_this_iter,
             step_ctx=train_iter_ctx,
-            iteration_results=train_results | eval_results,
+            iteration_results={**train_results, **eval_results},
```

Comment:
py3.8 does not support `dict | dict`.

Reply:
Isn't py3.8 deprecated by the next Ray release version?